### PR TITLE
Bootstrap-test: Added readme file

### LINKED ISSUE #3: 1
Please Kindly specify alternate text in the alt attribute as that provides alternative information for an image if a user for some reason cannot view it.

### PR DESCRIPTION
**Pull request by AlselwiAisha**
Added readme file

### CODE REVIEW

**Comment by erickma1** [comment]
Good professional outline of the readme file. However, you may consider working on the arrow icon in the drop-down menu of the "Tech Stack" section to display when clicked.

File: README.md [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+
+You can deploy this project using:
+
+<!--
+Example:
+
+```sh
+
+```
+ -->
```
Comment (MPDADDY) [inline README.md:128]: I think the your readme would be cleaner without this commented part .WHat do you think?

**Review by MPDADDY — COMMENTED** [review]
Good mit licence.

**Comment by ctokoli** [comment]
Please Kindly specify alternate text in the alt attribute as that provides alternative information for an image if a user for some reason cannot view it.

**Pull request closed without merging**